### PR TITLE
Update help documentation for the module

### PR DESCRIPTION
**Pull request by adityapatwardhan**
# PR Summary

Update help content for beta 19 version.

## PR Checklist

- [ ] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [ ] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [ ] None
    - **OR**
    - [ ] [Documentation needed]()
        - [ ] Issue filed: 
- **User-facing changes**
    - [ ] Not Applicable
    - **OR**
    - [ ] [Documentation needed]()
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [ ] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [ ] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and enumerated concerns in the summary.

### CODE REVIEW

File: help/Find-PSResource.md [hunk 1]
```diff
@@ -15,27 +15,23 @@ properties.
 ## SYNTAX
 
 ### ResourceNameParameterSet (Default)
-
 ```
-Find-PSResource [[-Name] <string[]>] [-Type <ResourceType>] [-Version <string>] [-Prerelease]
- [-Tag <string[]>] [-Repository <string[]>] [-Credential <pscredential>] [-IncludeDependencies]
- [<CommonParameters>]
+Find-PSResource [[-Name] <String[]>] [-Type <ResourceType>] [-Version <String>] [-Prerelease] [-Tag <String[]>]
+ [-Repository <String[]>] [-Credential <PSCredential>] [-IncludeDependencies] [<CommonParameters>]
```
Comment (alerickson) [inline help/Find-PSResource.md:20]: remove new line

File: help/Get-PSScriptFileInfo.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+external help file: PowerShellGet.dll-Help.xml
+Module Name: PowerShellGet
+online version:
+schema: 2.0.0
+---
+
+# Get-PSScriptFileInfo
+
+## SYNOPSIS
+{{ Fill in the Synopsis }}
```
Comment (PaulHigin) [inline help/Get-PSScriptFileInfo.md]: It looks like there are a few places like this, where new parameters information is missing (also with -SkipModuleManifestValidate).

**Review by PaulHigin — CHANGES_REQUESTED** [review]

File: help/Update-PSResource.md [hunk 3]
```diff
@@ -13,12 +13,10 @@ Downloads and installs the newest version of a package already installed on the
...
 
-### __AllParameterSets
-
 ```
-Update-PSResource [[-Name] <string[]>] [-Version <string>] [-Prerelease] [-Repository <string[]>]
- [-Scope <ScopeType>] [-TemporaryPath <string>] [-TrustRepository] [-Credential <pscredential>] [-Quiet] [-AcceptLicense]
- [-Force] [-PassThru] [-SkipDependencyCheck] [-AuthenticodeCheck] [-WhatIf] [-Confirm]
+Update-PSResource [[-Name] <String[]>] [-Version <String>] [-Prerelease] [-Repository <String[]>]
+ [-Scope <ScopeType>] [-TemporaryPath <String>] [-TrustRepository] [-Credential <PSCredential>] [-Quiet]
+ [-AcceptLicense] [-Force] [-PassThru] [-SkipDependencyCheck] [-AuthenticodeCheck] [-WhatIf] [-Confirm]
```
Comment (alerickson) [inline help/Update-PSResource.md:19]: Remove new line here

File: help/en-US/PowerShellGet.dll-Help.xml [hunk 4]
```diff
@@ -955,116 +955,227 @@ PSGallery        https://www.powershellgallery.com/api/v2       False         50
   </command:command>
   <command:command xmlns:maml="http://schemas.microsoft.com/maml/2004/10" xmlns:command="http://schemas.microsoft.com/maml/dev/command/2004/10" xmlns:dev="http://schemas.microsoft.com/maml/dev/2004/10" xmlns:MSHelp="http://msdn.microsoft.com/mshelp">
     <command:details>
-      <command:name>Install-PSResource</command:name>
-      <command:verb>Install</command:verb>
-      <command:noun>PSResource</command:noun>
+      <command:name>Get-PSResourceRepository</command:name>
```
Comment (alerickson) [inline help/en-US/PowerShellGet.dll-Help.xml]: This needs to be reverted

File: help/Update-ModuleManifest.md [hunk 5]
```diff
@@ -13,22 +13,19 @@ Updates a module manifest file.
...
+Update-ModuleManifest [-Path] <String> [-NestedModules <Object[]>] [-Guid <Guid>] [-Author <String>]
+ [-CompanyName <String>] [-Copyright <String>] [-RootModule <String>] [-ModuleVersion <Version>]
+ [-Description <String>] [-ProcessorArchitecture <ProcessorArchitecture>] [-CompatiblePSEditions <String[]>]
+ [-PowerShellVersion <Version>] [-ClrVersion <Version>] [-DotNetFrameworkVersion <Version>]
+ [-PowerShellHostName <String>] [-PowerShellHostVersion <Version>] [-RequiredModules <Object[]>]
+ [-TypesToProcess <String[]>] [-FormatsToProcess <String[]>] [-ScriptsToProcess <String[]>]
+ [-RequiredAssemblies <String[]>] [-FileList <String[]>] [-ModuleList <Object[]>]
+ [-FunctionsToExport <String[]>] [-AliasesToExport <String[]>] [-VariablesToExport <String[]>]
+ [-CmdletsToExport <String[]>] [-DscResourcesToExport <String[]>] [-Tags <String[]>] [-ProjectUri <Uri>]
+ [-LicenseUri <Uri>] [-IconUri <Uri>] [-ReleaseNotes <String>] [-Prerelease <String>] [-HelpInfoUri <Uri>]
```
Comment (alerickson) [inline help/Update-ModuleManifest.md:26]: remove new line here

File: help/Update-ModuleManifest.md [hunk 6]
```diff
@@ -13,22 +13,19 @@ Updates a module manifest file.
...
- [-RequireLicenseAcceptance] [-PrivateData <hashtable>] [<CommonParameters>]
+Update-ModuleManifest [-Path] <String> [-NestedModules <Object[]>] [-Guid <Guid>] [-Author <String>]
+ [-CompanyName <String>] [-Copyright <String>] [-RootModule <String>] [-ModuleVersion <Version>]
+ [-Description <String>] [-ProcessorArchitecture <ProcessorArchitecture>] [-CompatiblePSEditions <String[]>]
+ [-PowerShellVersion <Version>] [-ClrVersion <Version>] [-DotNetFrameworkVersion <Version>]
+ [-PowerShellHostName <String>] [-PowerShellHostVersion <Version>] [-RequiredModules <Object[]>]
+ [-TypesToProcess <String[]>] [-FormatsToProcess <String[]>] [-ScriptsToProcess <String[]>]
+ [-RequiredAssemblies <String[]>] [-FileList <String[]>] [-ModuleList <Object[]>]
+ [-FunctionsToExport <String[]>] [-AliasesToExport <String[]>] [-VariablesToExport <String[]>]
+ [-CmdletsToExport <String[]>] [-DscResourcesToExport <String[]>] [-Tags <String[]>] [-ProjectUri <Uri>]
```
Comment (alerickson) [inline help/Update-ModuleManifest.md:25]: remove new line

File: help/Update-ModuleManifest.md [hunk 7]
```diff
@@ -13,22 +13,19 @@ Updates a module manifest file.
...
- [-FormatsToProcess <string[]>] [-ScriptsToProcess <string[]>] [-RequiredAssemblies <string[]>]
- [-FileList <string[]>] [-ModuleList <Object[]>] [-FunctionsToExport <string[]>]
- [-AliasesToExport <string[]>] [-VariablesToExport <string[]>] [-CmdletsToExport <string[]>]
- [-DscResourcesToExport <string[]>] [-Tags <string[]>] [-ProjectUri <uri>] [-LicenseUri <uri>]
- [-IconUri <uri>] [-ReleaseNotes <string>] [-Prerelease <string>] [-HelpInfoUri <uri>] [-PassThru]
- [-DefaultCommandPrefix <string>] [-ExternalModuleDependencies <string[]>]
- [-RequireLicenseAcceptance] [-PrivateData <hashtable>] [<CommonParameters>]
+Update-ModuleManifest [-Path] <String> [-NestedModules <Object[]>] [-Guid <Guid>] [-Author <String>]
+ [-CompanyName <String>] [-Copyright <String>] [-RootModule <String>] [-ModuleVersion <Version>]
+ [-Description <String>] [-ProcessorArchitecture <ProcessorArchitecture>] [-CompatiblePSEditions <String[]>]
```
Comment (alerickson) [inline help/Update-ModuleManifest.md:19]: remove new line

File: help/Uninstall-PSResource.md [hunk 8]
```diff
@@ -14,17 +14,15 @@ Uninstalls a resource that was installed using **PowerShellGet**.
...
+Uninstall-PSResource [-Name] <String[]> [-Version <String>] [-Prerelease] [-SkipDependencyCheck]
  [-Scope <ScopeType>] [-WhatIf] [-Confirm] [<CommonParameters>]
 ```
 
 ### InputObjectParameterSet
-
 ```
-Uninstall-PSResource [-InputObject] <PSResourceInfo> [-Prerelease] [-SkipDependencyCheck]
- [-Scope <ScopeType>] [-WhatIf] [-Confirm] [<CommonParameters>]
+Uninstall-PSResource [-Prerelease] -InputObject <PSResourceInfo> [-SkipDependencyCheck] [-Scope <ScopeType>]
```
Comment (alerickson) [inline help/Uninstall-PSResource.md:24]: remove new line

File: help/Register-PSResourceRepository.md [hunk 9]
```diff
@@ -331,12 +295,23 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-### CommonParameters
+### -Force
+{{ Fill Force Description }}
```
Comment (alerickson) [inline help/Register-PSResourceRepository.md]: Fill out description

File: help/Publish-PSResource.md [hunk 10]
```diff
@@ -210,12 +210,27 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -SkipModuleManifestValidate
```
Comment (alerickson) [inline help/Publish-PSResource.md:213]: fill out description

File: help/PowerShellGet.md [hunk 11]
```diff
@@ -1,38 +1,33 @@
...
 
 ### [Get-PSResource](Get-PSResource.md)
 Returns modules and scripts installed on the machine via **PowerShellGet**.
 
 ### [Get-PSResourceRepository](Get-PSResourceRepository.md)
 Finds and returns registered repository information.
 
-### [Install-PSResource](Install-PSResource.md)
-Installs resources from a registered repository.
+### [Get-PSResourceRepository](Get-PSResourceRepository.md)
```
Comment (alerickson) [inline help/PowerShellGet.md]: this needs to be reverted

File: help/New-PSScriptFileInfo.md [hunk 12]
```diff
@@ -13,15 +13,12 @@ The cmdlet creates a new script file, including metadata about the script.
...
-New-PSScriptFileInfo [-Path] <string> -Description <string> [-Version <string>]
- [-Author <string>] [-Guid <guid>] [-CompanyName <string>] [-Copyright <string>]
- [-RequiredModules <hashtable[]>] [-ExternalModuleDependencies <string[]>]
- [-RequiredScripts <string[]>] [-ExternalScriptDependencies <string[]>] [-Tags <string[]>]
- [-ProjectUri <string>] [-LicenseUri <string>] [-IconUri <string>] [-ReleaseNotes <string>]
- [-PrivateData <string>] [-Force] [<CommonParameters>]
+New-PSScriptFileInfo [-Path] <String> [-Version <String>] [-Author <String>] -Description <String>
+ [-Guid <Guid>] [-CompanyName <String>] [-Copyright <String>] [-RequiredModules <Hashtable[]>]
+ [-ExternalModuleDependencies <String[]>] [-RequiredScripts <String[]>]
+ [-ExternalScriptDependencies <String[]>] [-Tags <String[]>] [-ProjectUri <String>] [-LicenseUri <String>]
```
Comment (alerickson) [inline help/New-PSScriptFileInfo.md:20]: remove new line

File: help/Install-PSResource.md [hunk 13]
```diff
@@ -6,545 +6,114 @@ online version:
 schema: 2.0.0
 ---
 
-# Install-PSResource
+# Get-PSResourceRepository
```
Comment (alerickson) [inline help/Install-PSResource.md]: file needs to be reverted

File: help/Get-PSScriptFileInfo.md (hunk 2 above)
Comment (alerickson) [inline help/Get-PSScriptFileInfo.md]: fill out contents

File: help/Find-PSResource.md [hunk 14]
```diff
@@ -15,27 +15,23 @@ properties.
...
  [<CommonParameters>]
 ```
 
 ### DscResourceNameParameterSet
-
 ```
-Find-PSResource -DscResourceName <string[]> [-Version <string>] [-Prerelease]
- [-ModuleName <string[]>] [-Tag <string[]>] [-Repository <string[]>] [-Credential <pscredential>]
- [-IncludeDependencies] [<CommonParameters>]
+Find-PSResource [-Version <String>] [-Prerelease] [-ModuleName <String[]>] -DscResourceName <String[]>
```
Comment (alerickson) [inline help/Find-PSResource.md:32]: remove new line

**Review by alerickson — CHANGES_REQUESTED** [review]
Just leaving notes of areas to fix

**Comment by alerickson** [comment]
Everything should be good to go now, except that 'help/en-US/PowerShellGet.dll-Help.xml' needs to be updated to reflect the new changes (I didn't want to make that change manually).

File: help/Get-PSScriptFileInfo.md (hunk 2 above)
Comment (adityapatwardhan) [reply help/Get-PSScriptFileInfo.md]: @alerickson fixed those. @PaulHigin Please re-review.

**Review by alerickson — APPROVED** [review]

**Review by PaulHigin — APPROVED** [review]

**Pull request merged**